packages/arb-util/configuration/configuration.go

```diff
@@ -788,6 +795,15 @@ func endCommonParse(k *koanf.Koanf) (*Config, *Wallet, error) {
 	wallet := out.Wallet
 	out.Wallet = Wallet{}
 
+	if out.FileInfo {
+		fmt.Printf("Database:         %s\n", out.Persistent.DatabasePath)
+		fmt.Printf("Database Backup:  %s\n", out.Core.SaveRocksdbPath)
+		fmt.Printf("Machine:          %s\n", out.Rollup.Machine.Filename)
+		fmt.Printf("Wallet DIrectory: %s\n", wallet.Local.Pathname)
+
+		os.Exit(0)
```

Comment:
Should we just always print this info instead of having a flag to print it?

Reply:
Updated it to print in plaintext and exit if flag is given, and do json log if log is not given.